File: API/Client/GetCampaignLoginsResponse.js
goog.provide('API.Client.GetCampaignLoginsResponse');

/**
 * Ответ на запрос списка логинов, связанных с магазином.
 * @record
 */
API.Client.GetCampaignLoginsResponse = function() {}

/**
 * Список логинов.
 * @type {!Array<!string>}
 * @export
 */
API.Client.GetCampaignLoginsResponse.prototype.logins;

File: src/main/scala/org/openapitools/client/api/GetCampaignLoginsResponse.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import GetCampaignLoginsResponse._

case class GetCampaignLoginsResponse (
  /* Список логинов. */
  logins: List[String])

object GetCampaignLoginsResponse {
  import DateTimeCodecs._

  implicit val GetCampaignLoginsResponseCodecJson: CodecJson[GetCampaignLoginsResponse] = CodecJson.derive[GetCampaignLoginsResponse]
  implicit val GetCampaignLoginsResponseDecoder: EntityDecoder[GetCampaignLoginsResponse] = jsonOf[GetCampaignLoginsResponse]
  implicit val GetCampaignLoginsResponseEncoder: EntityEncoder[GetCampaignLoginsResponse] = jsonEncoderOf[GetCampaignLoginsResponse]
}
